### PR TITLE
argocd 2.8.2

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by ZhongRuoyu — COMMENTED** [review]
> The formula version (2.8.2) is newer than the version from `brew livecheck` (2.6.14).

https://github.com/argoproj/argo-cd/releases

**Comment by p-linnane** [comment]
Looks like this is one of those scenarios where they are shipping multiple minor versions. v2.6.14 is marked as latest, but v2.8.2 is also released. We recently changed the livecheck because it was pulling tags before they were released, but this strategy also looks like it has it's own issues.

File: Formula/a/argocd.rb [hunk 1]
```diff
@@ -2,16 +2,19 @@ class Argocd < Formula
...
+  # There can be a notable gap between when a version is tagged and a
+  # corresponding release is created, so we check releases instead of the Git
+  # tags. Upstream maintains multiple major/minor versions and the "latest"
+  # release may be for an older version, so we have to check multiple releases
+  # to identify the highest version.
   livecheck do
     url :stable
-    strategy :github_latest
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+    strategy :github_releases
```
Comment (ZhongRuoyu) [inline Formula/a/argocd.rb:17]: This strategy should do. It's also used for [`grpc`](https://github.com/Homebrew/homebrew-core/blob/21c1ff76cbc6badb95d327d16258171b6dbba465/Formula/g/grpc.rb).

Comment (p-linnane) [reply Formula/a/argocd.rb:17]: Nice!

**Review by p-linnane — APPROVED** [review]

**Pull request merged**